int64 id
int64 cluster_id
float64 x
float64 y
float64 angle
